endpoints/openrtb2/auction_test.go

```diff
@@ -253,7 +253,12 @@ func (gr *getResponseFromDirectory) assert(t *testing.T) {
 
 		expectMsg := gr.messageGetter(t, fileData)
 		if len(expectMsg) > 0 {
-			assert.Equal(t, string(expectMsg), msg, "file %s had bad response body", filename)
+			//assert.Equal(t, string(expectMsg), msg, "file %s had bad response body", filename)
+			if string(expectMsg) != msg {
+				t.Errorf("[ERROR]     Test: %s \n", filename)
```

Comment:
We should rather use testify's assert package for assertions. It prints all this information out of the box

Reply:
rolled back the if statement. It was meant for debugging locally